presto-docs/src/main/sphinx/overview/concepts.rst

```diff
@@ -183,6 +183,35 @@ that statement. A query encompasses stages, tasks, splits, connectors,
 and other components and data sources working in concert to produce a
 result.
 
+Query Plan
+^^^^^^^^^^
+
+A query plan is a sequence of steps used to access and manipulate data 
+according to the SQL query. It is represented as a tree of nodes, with 
+each node loosely representing an `operator <https://prestodb.io/docs/current/overview/concepts.html#operator>`_. 
+It is represented as a tree of nodes, with each node representing a 
+logical unit of relational algebra that can be executed by the engine. 
```

Comment:
```suggestion
according to the SQL query. It is represented as a tree of nodes, with 
each node loosely representing an `operator <https://prestodb.io/docs/current/overview/concepts.html#operator>`_. 
```

It's a bit redundant, I think we just need the first sentence.

Reply:
Done! Screenshot of revised text in local doc build and pushed: 

<img width="792" alt="Screenshot 2024-10-16 at 11 17 13 AM" src="https://github.com/user-attachments/assets/f42796dc-cb01-4f82-9fb0-e9cadb0ff862">